src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -591,6 +596,8 @@ The returned values of your final tasks will be automatically printed.
 Finally, to have the hand on more parameters and features, the user should use ActiveEon Studio portals.
 The main ones are the _Resource Manager_, the _Scheduling Portal_ and the _Workflow Automation_.
 
+The example below show how the user can directly see his submitted job being executed in the scheduling portal:
```

Comment:
The example below shows how the user can directly display the running job in the scheduling portal: